bug-bounty-hof/client.yml

```diff
@@ -1,4 +1,74 @@
 names:
+- name: Sohom Datta
+  date: 2024-12-18
+- name: Narendra Bhati from Suma Soft Pvt. Ltd
+  date: 2024-12-18
+  twitter: "@imnarendrabhati"
+  url: https://twitter.com/@imnarendrabhati
+- name: Nan Wang
+  date: 2024-12-18
+  twitter: "@eternalsakura13"
+  url: https://twitter.com/@eternalsakura13
+- name: Dalmurino
+  date: 2024-12-11
+- name: Nils Bars
+  date: 2024-12-11
+- name: Irvan Kurniawan (sourc7)
+  date: 2024-12-04
+  url: https://www.linkedin.com/in/sourc7/
+- name: Paul Gerste
+  date: 2024-12-04
+- name: Jurrie Overgoor
+  date: 2024-11-27
+  url: https://www.linkedin.com/in/jurrieovergoor/
+- name: Dohyun Lee
+  date: 2024-11-20
+  twitter: "@l33d0hyun"
+  url: https://twitter.com/@l33d0hyun
+- name: Christian James
+  date: 2024-11-20
+- name: Lucas B. Williams
+  date: 2024-11-20
+- name: David & Johannes Henninger
+  date: 2024-11-20
+- name: Shaheen Fazim
+  date: 2024-11-14
+  twitter: "@shaheenfazim"
+  url: https://twitter.com/@shaheenfazim
+- name: Bharat (mrnoob)
+  date: 2024-11-14
+  url: https://www.linkedin.com/in/bharat-adhikari-726337225
+- name: P Umar Farooq
+  date: 2024-11-14
+- name: Masato Kinugawa
+  date: 2024-11-14
+- name: Renwa Hiwa
+  date: 2024-11-14
+- name: James Lee
+  date: 2024-11-13
+  twitter: "@Windowsrcer"
+  url: https://twitter.com/@Windowsrcer
+- name: Sun ZhiWei from Lenovo GIC security lab
+  date: 2024-11-13
+- name: Ali Syarief
+  date: 2024-11-01
+  url: https://www.linkedin.com/in/mohammad-ali-syarief/
+- name: Ronald Crane
+  date: 2024-10-23
+  url: https://www.zippenhop.com/
+- name: Damien Schaeffer
```

Comment:
Do we want to add ESET as affiliation? I think his colleague Romain Dumont was also credited in the blog post

Reply:
We only credit Damien in the CVE so I am just matching that